analyzers/its/expected/Net5/Net5--net5.0-S2589.json

```diff
@@ -4,6 +4,58 @@
 "id":  "S2589",
 "message":  "Change this condition so that it does not always evaluate to 'False'.",
 "location":  {
+"uri":  "https://github.com/SonarSource/sonar-dotnet/blob/master/analyzers/its/sources/Net5/Net5/Main.cs#L32",
+"region":  {
+"startLine":  32,
+"startColumn":  10,
+"endLine":  32,
+"endColumn":  13
+}
+}
+},
+{
+"id":  "S2589",
+"message":  "Change this condition so that it does not always evaluate to 'False'.",
+"location":  {
+"uri":  "https://github.com/SonarSource/sonar-dotnet/blob/master/analyzers/its/sources/Net5/Net5/Main.cs#L32",
+"region":  {
+"startLine":  32,
+"startColumn":  10,
+"endLine":  32,
+"endColumn":  13
+}
+}
+},
+{
+"id":  "S2589",
+"message":  "Change this condition so that it does not always evaluate to 'False'.",
+"location":  {
+"uri":  "https://github.com/SonarSource/sonar-dotnet/blob/master/analyzers/its/sources/Net5/Net5/Main.cs#L37",
```

Comment:
Why are we raising twice for each line? #L37 and #L32

Reply:
That's a good question. I need to investigae

Reply:
.NET 5 SDK had a bug. Some new nodes, like CompilationUnit due to top-level statement were called twice.
So we analyze it twice.

This only appears when you have global.json hardcoded .NET 5 SDK, or you have .NET 5 SDK as the latest SDK. It's fine if you have newer SDK and target net5.0.

I'll leave it like this.